packages/backend-output-schemas/src/auth/v1.ts

```diff
@@ -25,7 +25,7 @@ export const authOutputSchema = z.object({
 
     socialProviders: z.string().optional(), // JSON array as string
 
-    oauthDomain: z.string().optional(),
+    oauthCognitoDomain: z.string().optional(),
```

Comment:
should this be required now?

Reply:
It's only required if external providers are present. Since this is a flat config, we can't make it required.

Reply:
Ah right 👍